diff --git a/setup.py b/setup.py
index 1f0a58b..f7baa41 100644
--- a/setup.py
+++ b/setup.py
@@ -8,21 +8,8 @@ from pathlib import Path
 
 from setuptools import setup
 
-share = Path(sys.prefix, 'share')
-base = '/usr'
-if os.uname().sysname == 'Darwin':
-    base = '/usr/local'
-lib = Path(base, 'lib', 'password-store', 'extensions')
-
-if '--user' in sys.argv:
-    if 'PASSWORD_STORE_EXTENSIONS_DIR' in os.environ:
-        lib = Path(os.environ['PASSWORD_STORE_EXTENSIONS_DIR'])
-    else:
-        lib = Path.home() / '.password-store' / '.extensions'
-    if 'XDG_DATA_HOME' in os.environ:
-        share = Path(os.environ['XDG_DATA_HOME'])
-    else:
-        share = Path.home() / '.local' / 'share'
+share = Path('share')
+lib = Path('lib', 'password-store', 'extensions')
 
 setup(
     data_files=[
